Puzzlepart/did
# Adding surname, givenName and mail to exported timeentries

### Your checklist for this pull request

### Review checklist

### Description
Adding surname, givenName and mail to exported timeentries

### How to test
1. Check out locally with [gh](https://github.com/cli/cli)
2. Export time entries to Excel
3. Ensure columns are included

### Related issues
Closes #546
